doc/rbd/rbd-openstack.rst

```diff
@@ -252,15 +219,7 @@ Any OpenStack version except Mitaka
 
 If you want to enable copy-on-write cloning of images, also add under the ``[DEFAULT]`` section::
 
-    show_image_direct_url = True
-
-For Mitaka only
-^^^^^^^^^^^^^^^
-
-To enable image locations and take advantage of copy-on-write cloning for images, add under the ``[DEFAULT]`` section::
-
-    show_multiple_locations = True
-    show_image_direct_url = True
+show_image_direct_url = True
```

Comment:
Nit: missing indentation causing doc build error:

> /home/jenkins-build/build/workspace/ceph-pr-docs/doc/rbd/rbd-openstack.rst:222: WARNING: Literal block expected; none found.

Reply:
@dillaman I made the change :)